index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <nav>
+                <a href=""><img src="./assets/images/Logo Placeholder.png" alt=""></a>
+                <a href="http://"><img src="./assets/images/Enabled.png" alt="my logo"></a>
```

Comment:
- [ ] You've done an impressive job implementing the nav and anchor tags! 🎉😍
- [ ] However, using href='http://' creates an issue where clicking on the hamburger menu results in a blank page.
- [ ]  To resolve this, I recommend using href='#' or using only the `img` tag. Additionally, please ensure that you use appropriate alt attribute values to adhere to web accessibility guidelines and HTML best practices.🥱